recipe/meta.yaml

```diff
@@ -12,30 +12,21 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 0
+  number: 1
+  # These are present when the new environment is created
+  # so we have to exempt them from the list of initial files
+  # for conda-build to realize they should be included.
   entry_points:
     - conda = conda.cli:main
     - conda-env = conda_env.cli.main:main
   always_include_files:
```

Comment:
When we bump to conda 4.4, we'll probably have to update the whole recipe again here.  But at that point, we can just use what's in the conda.recipe directory in the repo.  [`bld.bat`](https://github.com/conda/conda/blob/4.4.x/conda.recipe/bld.bat) and ['build.sh`](https://github.com/conda/conda/blob/4.4.x/conda.recipe/build.sh) are just wrappers for the conda 4.4 recipe.  Just FYI for the future.